tests/test_prtvol2csv.py

```diff
@@ -139,6 +140,355 @@ def test_prtvol2csv(tmp_path, mocker):
             },
         }
     )
+    expected["FIPNAME"] = "FIPNUM"
+    pd.testing.assert_frame_equal(dframe, expected)
+
+
+def test_fipxxx(tmp_path, mocker):
+    """Test invocation from command line"""
+    prtfile = TEST_PRT_DATADIR / "DROGON_FIPZON.PRT"
+
+    os.chdir(tmp_path)
+
+    # Test for FIPZON:
+    with pytest.warns(FutureWarning, match="Output directories"):
+        mocker.patch(
+            "sys.argv", ["prtvol2csv", "--fipname", "FIPZON", "--debug", str(prtfile)]
+        )
+        prtvol2csv.main()
+    dframe = pd.read_csv("share/results/volumes/simulator_volume_fipnum.csv")
+
+    expected = pd.DataFrame.from_dict(
+        {
+            "FIPNUM": {0: 1, 1: 2, 2: 3},
+            "STOIIP_OIL": {0: 19549844.0, 1: 11560982.0, 2: 13683668.0},
+            "ASSOCIATEDOIL_GAS": {0: 102008.0, 1: 65583.0, 2: 6790.0},
+            "STOIIP_TOTAL": {0: 19651853.0, 1: 11626565.0, 2: 13690458.0},
+            "WIIP_TOTAL": {0: 153820626.0, 1: 136545137.0, 2: 166492503.0},
+            "GIIP_GAS": {0: 664018338.0, 1: 425072072.0, 2: 43915280.0},
+            "ASSOCIATEDGAS_OIL": {0: 2764802197.0, 1: 1647942723.0, 2: 1962726869.0},
+            "GIIP_TOTAL": {0: 3428820535.0, 1: 2073014795.0, 2: 2006642149.0},
+            "PORV_TOTAL": {0: 190002679.0, 1: 159669348.0, 2: 192088820.0},
+            "HCPV_OIL": {0: 28079757.0, 1: 16649101.0, 2: 19746072.0},
+            "WATPV_TOTAL": {0: 159065528.0, 1: 141192398.0, 2: 172153976.0},
+            "HCPV_GAS": {0: 2857395.0, 1: 1827848.0, 2: 188773.0},
+            "HCPV_TOTAL": {0: 30937151.0, 1: 18476950.0, 2: 19934844.0},
+        }
+    )
+    expected["FIPNAME"] = "FIPZON"
+    pd.testing.assert_frame_equal(dframe, expected)
+
+    # Test for FIPNUM:
+    mocker.patch(
+        "sys.argv", ["prtvol2csv", "--fipname", "FIPNUM", "--debug", str(prtfile)]
+    )
+    prtvol2csv.main()
+    dframe = pd.read_csv("share/results/volumes/simulator_volume_fipnum.csv")
+
+    expected = pd.DataFrame.from_dict(
+        {
+            "FIPNUM": {
```

Comment:
Is checking a smaller sample of these values sufficient to validate?

Reply:
Will discuss with Alif